app/feats/insurance_claim_feat.py

```
@@ -1,175 +1,561 @@
+                success=False,
+                error_code="MISSING_POLICY_UUID",
+                error_message="Granted event missing policy_uuid in payload",
+            )
+
+        try:
+            policy = StorePolicyResolver.resolve_store_item(policy_uuid)
+        except (PolicyNotFound, PolicyParseError, PolicyValidationError) as e:
+            return InsuranceClaimSubmissionResult(
+                success=False,
+                error_code="POLICY_NOT_RESOLVABLE",
+                error_message=f"Cannot resolve policy: {str(e)}",
```

Ruff (0.16.0)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)